package.json

```
@@ -6,7 +6,7 @@
 		"preview": "vite preview",
 		"build": "pnpm build:docs && pnpm build:package",
 		"build:docs": "svelte-kit sync && vite build",
-		"build:package": "svelte-package && publint",
+		"build:package": "svelte-kit sync && svelte-package && publint",
 		"publish": "pnpm build:package && changeset publish",
```

We've had annoying bugs in the past when we added a `publish` script since it interferes with the `npm publish` CLI command. Renaming it to something else fixes it.
```suggestion
		"ci:publish": "pnpm build:package && changeset publish",
```

renaming it to `release` works too if you prefer that as well

---

Thanks! Done!